strongbox-commons/src/main/java/org/carlspring/strongbox/config/PropertiesPathResolverConfig.java

```diff
@@ -0,0 +1,106 @@
+package org.carlspring.strongbox.config;
+
+import java.nio.file.Paths;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.core.env.Environment;
+import org.springframework.util.StringUtils;
+
+/**
+ * Configures a {@link PropertiesPathResolver} under the name "propsPath" - it can then be used in SpEL
+ * expressions to resolve properties lookup path's such as {@code @Value("#{propsPath.resolve('customPropName','defaultPropPath')}")}.
+ *
+ * @author cbono
+ */
+@Configuration
+class PropertiesPathResolverConfig
+{
+    @Bean(name = "propsPath")
```

Comment:
We typically don't abbreviate the names of beans like this. Would you mind using the full name (`propertiesPathResolver`)? Thanks! :)

Reply:
I usually don't either. However, the whole purpose of this bean is to be used in a SpEL statement so I strove for compact name. That being said, 

```java
@Value("#{@propsPath.resolve('strongbox.cron.tasks.yaml','etc/conf/strongbox-cron-tasks.yaml')}")
```
is not much better than 
```java
@Value("#{@propertiesPathBean.resolve('strongbox.cron.tasks.yaml','etc/conf/strongbox-cron-tasks.yaml')}")
```

I will expand it,

Reply:
@carlspring this is done.

Reply:
Yeah, I get what you mean... For brevity, it's better but for clarity -- probably not.

Thanks! :)